packages/gatsby-parallel-runner/src/processor-queue/implementations/google-functions/__tests__/index.js

```diff
@@ -0,0 +1,53 @@
+const { GoogleFunctions } = require(`..`)
+
+const googleConfig = { project_id: `test-project` }
+const processorSettings = { name: `image-processor`, key: `IMAGE_PROCESSOR` }
+
+test(`instantiate google pubsub`, async () => {
+  const pubSub = await new GoogleFunctions({
+    noSubscription: true,
+    processorSettings,
+    googleConfig,
+  })
+  expect(pubSub).toBeInstanceOf(GoogleFunctions)
+})
+
+test(`size check for google publish`, async () => {
+  const pubSub = await new GoogleFunctions({
+    noSubscription: true,
+    processorSettings,
+    googleConfig,
+  })
+  const msg = Buffer.from(`Hello, World!`)
+  pubSub.maxPubSubSize = 10000
+  pubSub.pubSubClient = {
+    topic: () => {
+      return {
+        publish: async msg => {
+          expect(msg).toBe(msg)
```

Comment:
Tautology?

Reply:
Good catch, fixed the shadowing that caused this